### PR TITLE
AST-3489 - Fix:Header Builder Submenu Animation Disrupts Keyboard Navigation.

### PR DESCRIPTION
**Pull request by yssambare12**
**Issue Description:**

Enabling animation for submenus in the Header builder interferes with keyboard navigation using the "Tab" and "Enter" buttons. This issue is not present when no animation is configured.

**Screen Recording:**

- https://d.pr/v/zPEwkV 

**Tickets:**

https://help.brainstormforce.com/conversation/25147?folder_id=183

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/css/unminified/menu-animation.css [hunk 1]
```diff
@@ -77,7 +79,8 @@
 .ast-desktop .astra-menu-animation-fade > .menu-item.focus > .sub-menu,
 .ast-desktop .astra-menu-animation-fade > .menu-item:hover > .sub-menu,
 .ast-desktop .astra-menu-animation-fade > .menu-item .menu-item.focus > .sub-menu,
-.ast-desktop .astra-menu-animation-fade > .menu-item .menu-item:hover > .sub-menu {
+.ast-desktop .astra-menu-animation-fade > .menu-item .menu-item:hover > .sub-menu,
+.ast-desktop .astra-menu-animation-fade > .menu-item > .sub-menu {
```
Comment (imnavanath) [inline assets/css/unminified/menu-animation.css:83]: @yssambare12 without any `:focus` , `:hover` this CSS required? This CSS can directly apply without any focus/hover action, can you please re-verify this, IMO we need to remove it.

**Comment by nathanrodrigues2111** [comment]
As mentioned in the call, @yssambare12 we need to also see that tabbing for sub menu also should work which is currently not working https://d.pr/v/LGPbeK

File: assets/css/unminified/menu-animation.css (hunk 1 above)
Comment (yssambare12) [reply assets/css/unminified/menu-animation.css]: @imnavanath Did some change in current implemention please check once.

**Comment by yssambare12** [comment]
> As mentioned in the call, @yssambare12 we need to also see that tabbing for sub menu also should work which is currently not working https://d.pr/v/LGPbeK

Handle this case as well with current CSS tweaks.

**Review by imnavanath — APPROVED** [review]

**Pull request merged**